src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -22,8 +22,7 @@ System.Runtime.CompilerServices.Unsafe</PackageDescription>
 
   <!-- Package servicing properties -->
   <PropertyGroup>
-    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">6.0.0.0</AssemblyVersion>
-    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">6.0.1.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true'">6.0.1.0</AssemblyVersion>
```

Comment:
For this one we should keep it pinned, because `netcoreapp2.1` where the package expects the framework to provide the implementation, is 
```
// C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\6.0.35\ref\net6.0\System.Runtime.CompilerServices.Unsafe.dll
// System.Runtime.CompilerServices.Unsafe, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
```

Reply:
Will bring back the two lines.